ui/src/assets/images/guardian-logo-320.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="320" height="60">
```

Comment:
In the new JS  rendering world for the website, are we keeping the same [image preparation rules](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/04-inline-svgs.md#image-preparation)? If so, the `xmlns` should be removed.